src/pages/Lobby.tsx

```diff
@@ -99,6 +97,19 @@ const LobbyPage = ({ ws, currentUser }: LobbyProps): JSX.Element => {
     }
   };
 
+  /**
+   * Remove game from games list.
+   * @param gameId string
+   */
+  const removeGameFromList = (gameId: string) => {
+    const newGames = [...games];
+    const index = games.findIndex((game) => game.gameId === gameId);
+    if (index !== 1) {
```

Comment:
do you mean not equal to `-1`? so u can find the game in the `games` array?
```
if (index !== -1) {
```

Reply:
Fixed